src/components/clusterWizard/wizardTransition.ts

```diff
@@ -14,9 +14,9 @@ export type ClusterWizardStepsType =
   | 'networking'
   | 'review';
 
-export const CLUSTER_WIZARD_FIRST_STEP: ClusterWizardStepsType = 'cluster-details';
+export const CLUSTER_WIZARD_FIRST_STEP: ClusterWizardStepsType = 'baremetal-discovery';
```

Comment:
This will not work for the "edit" flow.
I will try to fix it in a patch I am currently stacking on top of that PR.

Reply:
We could indicate the desired step via means of react router (state or query).